src/org/omegat/Bundle.properties

```diff
@@ -2332,7 +2332,12 @@ PROJECT_W_GLOSSARY=No writable glossary file in the glossary folder.
 PROJECT_LOCKED=Project locked. Perhaps it is already opened in another OmegaT instance
 PROJECT_INVALID=The specified folder is not an OmegaT project, or the project file is broken, or there is no valid backup.
 
-#Team work
+# Auto Saving exceptions
+AUTOSAVE_TIMEOUT_TAKING_LOCK=Lock trying timeout during autosave
+AUTOSAVE_GENERIC_ERROR=Got error during autosaving: {0}
+AUTOSAVE_INTERRUPTED=Save thread interrupted: {0}
+
```

Comment:
Questions and suggestions:

- `AUTOSAVE_TIMEOUT_TAKING_LOCK`
   What does "lock trying timeout" mean? What situation is it meant to describe?

- `AUTOSAVE_GENERIC_ERROR`
   --> "Error while autosaving: {0}" is a better phrasing, I think.

- `AUTOSAVE_INTERRUPTED`
   If this is a user facing message, it should just be "Autosaving interrupted: {0}"
   If it's only a log message for developers, it can remain as is.

Reply:
> AUTOSAVE_TIMEOUT_TAKING_LOCK
> What does "lock trying timeout" mean? What situation is it meant to describe?

I don't know. It is as same as text in source code.
All other messages are also as same as original.

Reply:
Original author is @alex73  
https://github.com/omegat-org/omegat/commit/dfa8097ac66df2e9b12d8acfca2aeced35511aa5

What's mean, @alex73  ?

Reply:
These message is raised on a code chunk here.

```
                   try {
                        Core.executeExclusively(false, () -> {
                            dataEngine.saveProject(false);
                            dataEngine.teamSyncPrepare();
                        });
                        Core.getMainWindow().showStatusMessageRB("ST_PROJECT_AUTOSAVED",
                                DateFormat.getTimeInstance(DateFormat.SHORT).format(new Date()));
                    } catch (TimeoutException ex) {
                        Log.logWarningRB("AUTOSAVE_TIMEOUT_TAKING_LOCK");
                    } catch (KnownException ex) {
                        Core.getMainWindow().showStatusMessageRB(ex.getMessage(), ex.getParams());
                    } catch (IRemoteRepository2.NetworkException ex) {
                        Log.logWarningRB("TEAM_NETWORK_ERROR", ex.getMessage());
                    } catch (OutOfMemoryError oome) {

                    } catch (Exception ex) {
                        Log.logWarningRB("AUTOSAVE_GENERIC_ERROR", ex.getMessage());
                    }
```

- `AUTOSAVE_TIMEOUT_TAKING_LOCK` is shown when `saveProject`  throws `TimeoutException` that may be rarely happened when user start saving manually and it had not finished before timeout duration.

Reply:
I've checked `Core.executeExclusively`  function  that is called  in the code chunk;

```
public static void executeExclusively(boolean waitForUnlock, RunnableWithException run)
            throws Exception {
        if (!EXCLUSIVE_RUN_LOCK.tryLock(waitForUnlock ? 180000 : 1, TimeUnit.MILLISECONDS)) {
            Exception ex = new TimeoutException("Timeout waiting for previous exclusive execution");
            Exception cause = new Exception("Previous exclusive execution");
            if (runningStackTrace != null) {
                cause.setStackTrace(runningStackTrace);
                ex.initCause(cause);
            }
            throw ex;
        }
```

The timeout exception will be thrown when multiple critical code are running and can not get `EXCLUSIVE_RUN_LOCK`.

Reply:
Yes, I noticed you simply moved the original hardcoded messages to `Bundle.properties`, which means they can be localized. It's probably still a good idea to improve on the phrasing when we can, especially when the original message was written by a non-native speaker doing their best.

(No slight on their efforts, which are highly appreciated. It's just very difficult to write in a non-native language.)

And speaking of localization, is keeping the FINE and FINER messages non-localized still relevant and appropriate? 

I don't know if @alex73 is still around. Does looking at the code give you an idea of what it accomplishes?

Some situations I can imagine (wild guesses):

If the autosave function locks the project temporarily while saving:
- Situation: The autosave function failed because its attempt lock the project timed out.

If it's the timeout function that gets locked:
- Situation: The attempt to lock the timeout function failed before the autosave completed.

Something else?

Reply:
> The timeout exception will be thrown when multiple critical code are running and can not get `EXCLUSIVE_RUN_LOCK`.

Looks like we ended up typing at the same time.

Thank you for investigating. I'll take a closer look later this evening.